addons/mail/static/tests/qunit_suite_tests/components/chat_window_manager_tests.js

```diff
@@ -968,12 +972,16 @@ QUnit.test('chat window: scroll conservation on toggle discuss', async function
         },
     });
 
-    await afterNextRender(() => messaging.discuss.open());
+    await openDiscuss();
     assert.containsNone(document.body, '.o_ChatWindow', "should not have any chat window after opening discuss");
 
     await afterEvent({
         eventName: 'o-component-message-list-scrolled',
-        func: () => messaging.discuss.close(),
+        func: () => openView({
+            res_id: mailChannelId1,
+            res_model: 'mail.channel',
+            views: [[false, 'form']],
+        }),
```

Comment:
Well it is because there is no such thing as just closing discuss. "Closing" discuss just means opening something else in terms of action manager. The easiest we can open in both community and enterprise is a form view like this.

Reply:
Ok, just nitpicking that test now relies more on other features than before, but this just shows that there is no such thing as just closing client action as you said.